requirements/app/base.txt

```diff
@@ -8,7 +8,7 @@ s3fs>=2022.5.0, <2022.8.3
 croniter>=1.3.0, <1.4.0  # strict; TODO: for now until we find something more robust.
 traitlets>=5.3.0, <=5.4.0
 arrow>=1.2.0, <1.2.4
-lightning-utilities==0.3.*
+lightning-utilities>=0.3.0, !=0.4.0, <0.5.0
```

Comment:
@Borda but we cannot allow installing `0.3` as we'll want to use functions that are only in 0.4.1

Reply:
What I don't understand is what blocks us from doing `==0.4.1`

Isn't the app dependency pulled from this file?

Reply:
the problem is dependencies in other ecosystem packages like Bolts, Flash so they fail for no common versions, see for example: https://github.com/Lightning-AI/lightning-bolts/issues/937

Reply:
```suggestion
lightning-utilities>=0.3.*, !=0.4.0, <0.5.0
```